src/elements/content-explorer/ContentExplorer.js

```diff
@@ -290,6 +296,13 @@ class ContentExplorer extends Component<Props, State> {
         } else {
             this.fetchFolder(currentFolderId);
         }
+
+        if (isFeatureEnabled(features, 'contentExplorer.gridView.enabled')) {
+            const pref = this.store.getItem(localStoreViewMode);
+            if (pref && (pref === VIEW_MODE_GRID || pref === VIEW_MODE_LIST)) {
+                this.changeViewMode(pref);
```

Comment:
This will call `setState` after the component is mounted, which will result in a double render. We may be able to eliminate this by calling `getItem` during render.